tests/ios/Podfile

```diff
@@ -5,7 +5,7 @@ appPackage = JSON.parse(File.read(File.join('..', '..', 'node_modules', '@react-
 $FirebaseSDKVersion = appPackage['sdkVersions']['ios']['firebase']
 #$FirebaseSDKVersion = '7.7.0' # https://firebase.google.com/support/release-notes/ios
 Pod::UI.puts "react-native-firebase/tests: Using Firebase SDK version '#{$FirebaseSDKVersion}'"
-$RNFirebaseAsStaticFramework = false # toggle this to true (and set 'use_frameworks!' below to test static frameworks)
+#$RNFirebaseAsStaticFramework = false # toggle this to true (and set 'use_frameworks!' below to test static frameworks)
```

Comment:
this was commented out because false is the default, the comment is still clear, and with it set it doubled the amount of pod install output, obscuring other things that were maybe non-default